.dockerignore

```diff
@@ -1,6 +0,0 @@
-.cosa
```

Comment:
Shouldn't be deleted

Reply:
I'll update that, I was experimenting with some building so it's a bit hacky right now. Thanks for the spot